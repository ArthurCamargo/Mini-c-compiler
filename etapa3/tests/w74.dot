digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 -> n4;
n2 -> n5;
n4;
n4 [label="c"];
n5;
n5 [label="5"];
n2 [label="[]"];
n3;
n3 -> n6;
n3 -> n7;
n6;
n6 -> n8;
n6 -> n9;
n8;
n8 [label="3"];
n9;
n9 -> n10;
n10;
n10 [label="a"];
n9 [label="f"];
n6 [label="*"];
n7;
n7 [label="a"];
n3 [label="+"];
n1 [label="="];
n0 [label="f"];

}