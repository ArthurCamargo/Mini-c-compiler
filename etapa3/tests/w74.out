0x55b31ebc03c0
0x55b31ebc03c0, 0x55b31ebc0840
0x55b31ebc0840
0x55b31ebc0840, 0x55b31ebc05b0
0x55b31ebc0840, 0x55b31ebc07f0
0x55b31ebc05b0
0x55b31ebc05b0, 0x55b31ebc0510
0x55b31ebc05b0, 0x55b31ebc0560
0x55b31ebc0510
0x55b31ebc0510 [label="c"];
0x55b31ebc0560
0x55b31ebc0560 [label="5"];
0x55b31ebc05b0 [label="[]"];
0x55b31ebc07f0
0x55b31ebc07f0, 0x55b31ebc0730
0x55b31ebc07f0, 0x55b31ebc07a0
0x55b31ebc0730
0x55b31ebc0730, 0x55b31ebc0600
0x55b31ebc0730, 0x55b31ebc0670
0x55b31ebc0600
0x55b31ebc0600 [label="3"];
0x55b31ebc0670
0x55b31ebc0670, 0x55b31ebc06e0
0x55b31ebc06e0
0x55b31ebc06e0 [label="a"];
0x55b31ebc0670 [label="f"];
0x55b31ebc0730 [label="*"];
0x55b31ebc07a0
0x55b31ebc07a0 [label="a"];
0x55b31ebc07f0 [label="+"];
0x55b31ebc0840 [label="="];
0x55b31ebc03c0 [label="f"];
